### PR TITLE
Post about Azure Emulator on Openstack

### PR DESCRIPTION
**Pull request by andricDu**
@Fgerthoffert @btiernay

### CODE REVIEW

File: _posts/dusan_andric/2016-10-28-azure-openstack.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+## The How-To
+
+### Openstack & Windows
+
+Those of you running an Openstack cloud are probably wondering how this is going to work. Thankfully a company 
+by the name of Cloudbase Solutions has solved this problem. They specialize in bringing the worlds of Openstack and Windows together. 
+Even better they provide a ready to use Windows Server 2012 Evaluation guest image. 
+
+Go ahead and get the image from here: https://cloudbase.it/windows-cloud-images/
```
Comment (cheapsteak) [inline _posts/dusan_andric/2016-10-28-azure-openstack.md]: Looks like Kramdown doesn't auto convert things that look like links to links

Might as well do this then？

```
[get the image from here](https://cloudbase.it/windows-cloud-images/)
```

**Comment by andricDu** [comment]
@shanewilson

File: _posts/dusan_andric/2016-10-28-azure-openstack.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+If your daily driver is a Mac, I highly recommend downloading Microsoft Remote Desktop for working with the Windows instance.
+You can find it on the App Store. 
+
+### Azure Setup
+
+First thing you'll want to install is SQL Server Express LocalDB. It is required by the Storage Emulator. [You can find it here.](https://www.microsoft.com/en-us/sql-server/sql-server-editions-express)
+
+Next, download the standalone installer for the emulator by following the link in the [Azure Storage Emulator documentation.](https://azure.microsoft.com/en-us/documentation/articles/storage-use-emulator/)
+
+Once everything is installed you can start the storage emulator by searching for it from the start screen. It should open up a new cmd window. You can this verify that it is running:
```
Comment (Fgerthoffert) [inline _posts/dusan_andric/2016-10-28-azure-openstack.md]: You can then verify ... ?

File: _posts/dusan_andric/2016-10-28-azure-openstack.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+And there it is:
+
+<figure>
+    <img src="{{site.urlimg}}dusan_andric/azure/download.png" />
+    <figcaption>Successful download of image.</figcaption>
+</figure>
+
+## Conclusion
+
+I think it is safe to say that Microsoft products and services are no longer scary when coming from a Unix environment.
```
Comment (Fgerthoffert) [inline _posts/dusan_andric/2016-10-28-azure-openstack.md]: Well, a fair bit of this article explains a "workaround" to connect. I would call it no longer scary once Microsoft implement a "no-workaround" solution :p

Note: This comment is a troll ...

**Comment by andricDu** [comment]
Thanks for the feedback!

**Comment by Fgerthoffert** [comment]
Great additions @andricDu

**Pull request merged**